.tests/cypress/integration/mini-cart/add-to-cart-configurable.js

```diff
@@ -1,85 +1,82 @@
-                'X-Magento-Vary',
-                'permission-profiling',
-                'PHPSESSID',
-                'form_key',
-            ]
-        })
+    // Hide premissions popup
+    cy.setCookie('permission-cookies', 'true')
+    cy.setCookie('permission-profiling', 'true')
+    cy.setCookie('mage-cache-sessid', 'true')
+    // Keep cookies beween tests
+    Cypress.Cookies.defaults({
+      whitelist: [
+        'frontend',
+        'X-Magento-Vary',
+        'permission-profiling',
+        'PHPSESSID',
+        'form_key',
+      ]
     })
+  })
 
-    it('Visits product', () => {
-        cy.visit('https://alpaca-ce-solr-demo.snowdog.pro/configurable-product')
-        cy.get('.breadcrumbs__list').should('be.visible')
-    })
+  it('Visits product', () => {
+    cy.visit('https://alpaca-ce-solr-demo.snowdog.pro/configurable-product')
+    cy.get('.breadcrumbs__list').should('be.visible')
+  })
 
-    it('Check visiblility of content', () => {
-        cy.get('.product-view__main-details').should('be.visible')
-        cy.get('[data-ui-id=page-title-wrapper]').should('be.visible')
-        cy.get('.product-view__information').should('be.visible')
-        cy.get('.product-view__sku').should('be.visible')
-        cy.get('#product-price-2048').should('be.visible')
-        cy.get('.quantity-update').should('be.visible')
-        cy.get('#product-addtocart-button').should('be.visible')
-        cy.get('.product-view__extra-actions').should('be.visible')
-        cy.get('[data-testid=product-tab__title]').contains('Details').should('be.visible')
-        cy.get('[data-testid=product-tab__title]').contains('Reviews').should('be.visible')
+  it('Check visiblility of content', () => {
+    cy.get('.product-view__main-details').should('be.visible')
+    cy.get('[data-ui-id=page-title-wrapper]').should('be.visible')
+    cy.get('.product-view__information').should('be.visible')
+    cy.get('.product-view__sku').should('be.visible')
+    cy.get('#product-price-2048').should('be.visible')
+    cy.get('.quantity-update').should('be.visible')
+    cy.get('#product-addtocart-button').should('be.visible')
+    cy.get('.product-view__extra-actions').should('be.visible')
+    cy.get('[data-testid=product-tab__title]').contains('Details').should('be.visible')
+    cy.get('[data-testid=product-tab__title]').contains('Reviews').should('be.visible')
 
-    })
+  })
 
-    it('Test reviews tab', () => {
-        cy.get('[data-testid=product-tab__title]').contains('Reviews').click()
-        cy.get('.product-review__title').contains('Customers reviews')
-        cy.get('#review-form').should('be.visible')
-        cy.get('#reviewAddButton')
-    })
-    it('Checks configurable options', () => {
+  it('Test reviews tab', () => {
+    cy.get('[data-testid=product-tab__title]').contains('Reviews').click()
+    cy.get('.product-review__title').contains('Customers reviews')
+    cy.get('#review-form').should('be.visible')
+    cy.get('#reviewAddButton')
```

Comment:
Thee should be a chaining action to this yielded element. For example it should be visible.